src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicPlatformDetectionTests.cs

```diff
@@ -17,8 +16,8 @@ public class MsQuicPlatformDetectionTests : QuicTestBase<MsQuicProviderFactory>
         [ConditionalFact(nameof(IsQuicUnsupported))]
         public void UnsupportedPlatforms_ThrowsPlatformNotSupportedException()
         {
-            Assert.Throws<PlatformNotSupportedException>(() => CreateQuicListener());
-            Assert.Throws<PlatformNotSupportedException>(() => CreateQuicConnection(new IPEndPoint(IPAddress.Loopback, 0)));
+            Assert.ThrowsAsync<PlatformNotSupportedException>(async () => await CreateQuicListener());
+            Assert.ThrowsAsync<PlatformNotSupportedException>(async () => await CreateQuicConnection(new IPEndPoint(IPAddress.Loopback, 0)));
```

Comment:
This should still work, right?

```suggestion
            Assert.ThrowsAsync<PlatformNotSupportedException>(() => CreateQuicListener());
            Assert.ThrowsAsync<PlatformNotSupportedException>(() => CreateQuicConnection(new IPEndPoint(IPAddress.Loopback, 0)));
```

Reply:
It's `ValueTask` vs `Task` problem. So either async await combo, or `AsTask`. I don't care which way.